src/haddock/clis/cli_score.py

```diff
@@ -170,7 +171,8 @@ def main(
         if param not in default_emscoring:
             sys.exit(
                 f"* ERROR * Parameter {param!r} is not a "
-                "valid `emscoring` parameter"
+                f"valid `emscoring` parameter.{os.linesep}"
+                f"Valid emscoring parameters are: {list(default_emscoring.keys())}"
```

Comment:
instead of `{list(default_emscoring.keys())}`, I would put `sorted(default_emscoring)` for order reproducibility.
Also some kind of `', '.join(sorted(default_emscoring))` to have it nicely displayed ?

Reply:
good point, I'll implement this